x/tokenfactory/types/params.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// NewParams creates a new params instance
+func NewParams(denomCreationFee sdk.Coins) Params {
+	return Params{
+		DenomCreationFee: denomCreationFee,
+	}
+}
+
+// DefaultParams creates a default params instance
+func DefaultParams() Params {
+	return Params{
+		DenomCreationFee: sdk.NewCoins(sdk.NewInt64Coin(sdk.DefaultBondDenom, 10_000_000)), // 10 DSM
```

Comment:
We should increase this significantly. I believe setting a default value of 10,000 DSM would be better in this case. Users will always have the option to request a community fund for this purpose, and we can adjust it later if needed. However, starting with a 10,000 DSM fee appears to be the most favorable choice to ensure a substantial token burn.

Reply:
Fixed